### DISCUSSION TITLE
Does anyone have examples of Livewire paginated pages used in production?

### DISCUSSION THREAD

**Discussion by tinusg** [livewire/livewire#5506] [1. Help]
Does anyone have an online example of Livewire paginated pages in production so I can check an issue I have with them in Safari?

I have unexpected results with the back button on Livewire paginated pages. 

I built an image gallery with 3 pages, using Livewire pagination. When I navigate to page 3, visit one of the images (on its own url) and go back using the back button, page 3 loads ok. If I then click to page 2, click on of the images again and use te back button, I see page 3, even though the URL says ?page=2.

It works fine in Chrome, which leads me to believe it's an issue with Safari.

**Comment by joshhanley** [comment]
@tinusg no, sorry. But can you make a test Laravel app to test it out yourself?

**Comment by EskimoEmoji** [comment]
@tinusg I am having a very similar issue. I am using pagination on my 'Home' page. I click on an item that goes to the 'item' page. When I hit the safari back button. The 'home' page shows for a fraction of a second and then goes back to the 'item' page. This seems to happen about 50% of the time on a Safari desktop browser and 100% of the time in a safari mobile browser. I don't have this issue with any other browser.

I added the below code that helped desktop some but doesn't help mobile at all. This is a really annoying issue. Have you had any luck?
https://github.com/livewire/livewire/issues/2114#issuecomment-766530813
<head>
    <script>
        (function() {
            window.onpageshow = function(event) {
                if (event.persisted) {
                    window.location.reload();
                }
            };
        })();
    </script>
    
</head>

**Reply by EskimoEmoji** [reply]
It definitely is a pagination issue. when I comment out 'use WithPagination;' the issue is gone. but I can't go to next page

**Reply by tinusg** [reply]
I decided to not use Livewire pagination in cases where I want the back button to be reliable.
